### ISSUE TITLE
Cannot find type definition file for 'lodash' error on following step described in npm site.

### ISSUE THREAD

**Issue by RohitRane** [scttcper/toastr-ng2#20]
I followed all the steps listed in :   https://www.npmjs.com/package/toastr-ng2

I am using the fountain.js scaffolding with webpack for my angular 2 project. 

I am getting the following typescript error hence my code is not compiling further :  
```
ERROR in /project-root/node_modules/toastr-ng2/portal/portal.d.ts
(1,1): error TS2688: Cannot find type definition file for 'lodash'.
```

I went ahead and even did npm i lodash, and typings install lodash and 
import * as _ from 'lodash'; in my project but nothing helped.
 
Please help !!

**Comment by scttcper** [comment]
Sounds like a release issue. I'll come up with a fix soon. You can also try installing `@types/lodash` to fix now.

**Comment by scttcper** [comment]
fixed in 1.5.6
